include/variables.h

```diff
@@ -46,9 +46,13 @@ extern OSViContext* __osViNext;
 extern OSViMode osViModeFpalLan1;
 extern OSViMode gCustomViModePal60Lan1;
 extern u32 __additional_scanline;
-extern const char gBuildMakeOption[];
-extern const char gBuildGitVersion[];
-extern const char gBuildTeam[];
+#if IS_MAP_SELECT_ENABLED
```

Comment:
``IS_DEBUG_BOOT_ENABLED``? also some of these might be used on the rotating n64 logo screen (like the build author and the date)